### PR TITLE
OKD: Add Virtualization documentation

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Making virtualization related documentation available on OKD as well.
Ensuring that generated pages references correctly OKD Virtualization and not OpenShift Virtualization.

This patch is proposed following discussion within [OKD Virtualization SIG](https://okd-virtualization.github.io/)
    
Signed-off-by: Sandro Bonazzola <<sbonazzo@redhat.com>>

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
CC: @fabiand , @tiraboschi , @michalskrivanek

**Comment by sandrobonazzola** [comment]
@stoobie @emarcusRH @rolfedh any clue on why this is failing CI while building fine locally?

**Comment by rolfedh** [comment]
@sandrobonazzola Travis CI (https://app.travis-ci.com/github/openshift/openshift-docs/builds/238152680) is complaining:
```
>>> Working on virt book in openshift-enterprise <<<
Transforming the AsciiDoc content to DocBook XML...
Unknown ID or title "preparing-cluster-for-virt", used as an internal cross reference
Unknown ID or title "virt-collecting-virt-data", used as an internal cross reference
Unknown ID or title "installing-virt-web", used as an internal cross reference
Unknown ID or title "installing-virt-cli", used as an internal cross reference
Unknown ID or title "virt-collecting-virt-data", used as an internal cross reference
Unknown ID or title "installing-virt-cli", used as an internal cross reference
Unknown ID or title "installing-virt-web", used as an internal cross reference
Unknown ID or title "virt-specifying-nodes-for-virtualization-components", used as an internal cross reference
Unknown ID or title "virt-specifying-nodes-for-virtualization-components", used as an internal cross reference
Unknown ID or title "virt-specifying-nodes-for-virtualization-components", used as an internal cross reference
>>> Validation of book virt in openshift-enterprise failed <<<
>>> Finished with virt book in openshift-enterprise <<<
```
This usually happens when cross-references point to content that has recently been removed. The CI does not allow broken links like this to exist.

**Comment by sandrobonazzola** [comment]
About: Unknown ID or title "preparing-cluster-for-virt", used as an internal cross reference

```
$ find . -name "preparing-cluster-for-virt*"
./virt/install/preparing-cluster-for-virt.adoc
```
and

```
$ LC_ALL=C grep -ri preparing-cluster-for-virt .
Binary file ./.git/index matches
./.git/COMMIT_EDITMSG:#	modificato:             virt/install/preparing-cluster-for-virt.adoc
./modules/virt-cluster-resource-requirements.adoc:// * virt/install/preparing-cluster-for-virt.adoc
./virt/install/preparing-cluster-for-virt.adoc:[id="preparing-cluster-for-virt"]
./virt/install/preparing-cluster-for-virt.adoc::context: preparing-cluster-for-virt
./virt/virt-learn-more-about-openshift-virtualization.adoc:| xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[Configuring your cluster for {VirtProductName}]
./_topic_map.yml:    File: preparing-cluster-for-virt
./_topic_map.yml:    File: preparing-cluster-for-virt
```
And within _topic_map.yml preparing-cluster-for-virt looks correctly nested within virt/install

I don't see why it should fail.

I'll check the other too but sounds suspicious.

**Comment by sandrobonazzola** [comment]
maybe I got it.
`./virt/virt-learn-more-about-openshift-virtualization.adoc` is missing inclusion of  `virt/install/preparing-cluster-for-virt.adoc` so the xref is missing in the single html file generated from master.adoc during the conversion to DocBook XML.
But it's not a change I introduced so I have no clue on how it could pass CI before.

**Comment by sandrobonazzola** [comment]
Nevermind, found the issue, pushing a fix.

**Comment by ousleyp** [comment]
Hi @sandrobonazzola, thank you for this submission. The CNV docs team needs to do some research on our end before we can consider merging this. Holding the PR in the meantime.

**Comment by fabiand** [comment]
@ousleyp what needs to be done? Maybe we can help a little from the dev side here.

**Comment by ousleyp** [comment]
> @ousleyp what needs to be done? Maybe we can help a little from the dev side here.

Hi Fabian, sorry for the delay. We've been getting acquainted with how the OpenShift core docs team works with OKD. I think there are just a couple of things to agree on before we move forward. Let's discuss this in chat. :)

**Comment by sandrobonazzola** [comment]
Rebased as requested

File: _topic_map.yml [hunk 1]
```diff
@@ -2762,37 +2762,81 @@ Topics:
   - Name: Removing Jaeger
     File: rhbjaeger-removing
 ---
-Name: OpenShift Virtualization
+Name: OpenShift/OKD Virtualization
```
Comment (fabiand) [inline _topic_map.yml:2765]: I fear we have to solve this before bothering anybody to get an ack

Comment (sandrobonazzola) [reply _topic_map.yml:2765]: Ok, I'm on it.

**Pull request closed without merging**